editoast/README.md

```diff
@@ -90,6 +90,41 @@ Running editoast with deactivated cache can help repeating calls when debugging.
 NO_CACHE=true cargo run -- runserver
 ```
 
+## Authorization
+
+
+### How to disable authorizations
+
+By default editoast is running with authorization enabled. 
+You can disable it by using the env variable `EDITOAST_ENABLE_AUTHORIZATION=false`
+
+```sh 
+EDITOAST_ENABLE_AUTHORIZATION=false cargo run -- runserver
+```
+
+If you run the stack with docker : 
+
+```sh 
+EDITOAST_ENABLE_AUTHORIZATION=false docker compose up  
+```
+
+### User & role management
+
+You can create a new user with `Admin` role by using the following commands : 
+
+```sh 
+cargo run user add 'mock/mocked' 'Example User'
+cargo run roles add 'mock/mocked' Admin 
+```
+
+Where `mock/mocked` is the **identity** of the user and `Example User` its **name**. 
```

Comment:
It may be worth mentioning that the `mock/mocked` key comes from the gateway mocking configuration?